scripts/gulpfiles/release_notes.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Gulp scripts for creating release notes.
+ */
+
+var gulp = require('gulp');
+var fetch = require('node-fetch');
+
+const baseUrl = "https://api.github.com/graphql";
```

Comment:
[Style guide for constants](https://google.github.io/styleguide/jsguide.html#naming-constant-names): `BASE_URL`.